### PR TITLE
Merge ilastik and lazyflow repos

### PR DESCRIPTION
**Pull request by m-novikov**
Merge ilastik and lazyflow histories

### CODE REVIEW

**Comment by m-novikov** [comment]
4 failing tests:
- Singleton weirdness should be converted to `Singleton.getInstance` to fix.
- Logging is messed up by top-level logging configuration in ilastik test files.
- pytest-qt installs top level hooks that prevent bubbling up of some exceptions

**Comment by m-novikov** [comment]
If we decide to proceed we would need:
1. PR to ilastik moving test files and fixing issues with `logging`, `CacheManager` and disabling `pytest-qt` exception hook
2. PR to lazyflow moving test files and docs and deleting some conflicting files like `CONTRIBUTING.md` and adding `CacheManager.getInstance` method instead of `Signletion` metaclass
3. Proper PR from ilastik bot to merge repos

**Comment by k-dominik** [comment]
@akreshuk: how about you merge this one, to give it even more gravity :)

File: .gitignore [hunk 1]
```diff
@@ -72,3 +73,11 @@ tests/recorded_test_cases/ObjectClassification/cube_objects_raw.npy
...
 *.swp
 docs/**/_build
+
+.coverage
+*.backup
+*.*~
+*.jpg
+*.swp
+
+lazyflow/drtile/drtile.pyd
```
Comment (k-dominik) [inline .gitignore]: drtile is gone, should have been also removed from .gitignore
```suggestion
```

**Comment by akreshuk** [comment]
OMG, it's happening for real!

**Pull request merged**